<launch>
  <!-- LKAS -->
  <node pkg="limo_drive" type="lkas_node.py" name="lkas" output="screen" />

  <!-- Detect (obstacle_state) -->
  <node pkg="limo_drive" type="obstacle_detect_node.py" name="obstacle_detect" output="screen" />

  <!-- 장애물 회피 노드(DWA) 켜기 -->
  <node pkg="limo_drive"
        type="obstacle_avoid_node.py"    
        name="mission3_dwa"
        output="screen">
    <!-- 토픽 이름은 지금 코드 디폴트랑 맞춰줌 -->
    <param name="scan_topic"   value="/scan" />
    <param name="odom_topic"   value="/odom" />
    <param name="cmd_topic"    value="/cmd_vel_obstacle" />
    <param name="enable_topic" value="/obstacle_enable" />
  </node>

  <!-- FSM / MUX -->
  <node pkg="limo_drive" type="fsm_mux_node.py" name="fsm_mux" output="screen" />
</launch>

